src/pages/gateway/fastly.md

```diff
@@ -0,0 +1,257 @@
+---
+title: Fastly 
+description: Learn how to configure a Fastly CDN for Adobe Commerce using API Mesh.
+keywords:
+  - API Mesh
+  - Extensibility
+  - Cache
+  - GraphQL
+  - Integration
+  - REST
+  - Tools
+---
+
+# Fastly configuration
+
+Using [Fastly](https://experienceleague.adobe.com/docs/commerce-cloud-service/user-guide/cdn/fastly.html?lang=en) as a content delivery network (CDN) with API Mesh for Adobe Developer App Builder provides additional security and improved performance. To fully integrate API Mesh, Fastly, and Adobe Commerce, you must follow the configuration instructions in the following sections.
+
+## Configure Fastly in Adobe Commerce
+
+Use the following process to configure edge caching for Adobe Commerce using the provided Fastly CDN.
```

Comment:
edge caching should be here dynamic content - as it's not based on the headers that API mesh outputs like explained in [cache-control-headers.md](https://github.com/AdobeDocs/graphql-mesh-gateway/pull/161/files#diff-c2d1334ee9d83dd922562bc477db7e2f8a72e8c241f1f9be050cb3a7b6ef0f7f)

edit:
following steps process not process